src/components/Wizard/CreateVmWizard/providers/index.js

```diff
@@ -0,0 +1,44 @@
+import { get } from 'lodash';
+
+import { NAMESPACE_KEY, PROVIDER_KEY, PROVIDER_VMWARE } from '../constants';
+import { isImageSourceType, settingsValue } from '../../../../k8s/selectors';
+import { PROVISION_SOURCE_IMPORT } from '../../../../constants';
+
+import { getVMWareSection } from './vmware';
+import { startV2VVMWareController } from './v2vvmwareController';
+import { HELP_PROVIDER_VMWARE } from '../strings';
+
+const providerList = [PROVIDER_VMWARE];
+
+const getProviderHelp = basicSettings => {
+  const provider = settingsValue(basicSettings, PROVIDER_KEY);
+  switch (provider) {
+    case PROVIDER_VMWARE:
+      return HELP_PROVIDER_VMWARE;
+    default:
+      return null;
+  }
+};
+
+export const importProviders = (basicSettings, WithResources, k8sCreate, k8sGet, k8sPatch) => ({
+  [PROVIDER_KEY]: {
+    id: 'provider-dropdown',
+    title: 'Provider',
+    type: 'dropdown',
+    defaultValue: '--- Select Provider ---',
+    choices: providerList,
+    required: true,
+    onChange: (...opts) => onProviderChanged(k8sCreate, k8sGet, ...opts),
+    isVisible: basicVmSettings => isImageSourceType(basicVmSettings, PROVISION_SOURCE_IMPORT),
+    help: getProviderHelp(basicSettings),
+  },
+  ...getVMWareSection(basicSettings, WithResources, k8sCreate, k8sGet, k8sPatch),
+});
+
+const onProviderChanged = (k8sCreate, k8sGet, valueValidationPair, key, formValid, prevBasicSettings) => {
+  startV2VVMWareController({ k8sCreate, k8sGet, namespace: get(prevBasicSettings[NAMESPACE_KEY], 'value') });
```

Comment:
if I do not select namespace before `startV2VVMWareController` is called, it will fail. Latest version of Create VM Wizard Design https://docs.google.com/document/d/1ks4H-ImD-r0s5eV4FsP_73kRdAP6sKJTRbj8nClpyp0/edit#heading=h.o4npgyz4fz5s guides user also by disabling fields we are not 'ready' to be selected. Ie when I open wizard, only name/description/namespace should be enabled. Once I fill those, it will enable Provision Source and so on. We should implement it soon.

Reply:
Follow up issue created https://github.com/kubevirt/web-ui-components/issues/239

Reply:
Good catch, I'll fix that.